karma.conf.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function (config) {
+    config.set({
+        
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: './',
+        
+        // frameworks to use
+        // some available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+        frameworks: ['jasmine'],
+        
+        // list of files / patterns to load in the browser
+        files: [
+            './node_modules/angular/angular.js',
+            './node_modules/angular-mocks/angular-mocks.js',
+
+            './src/javascript/app/ng-wig/ng-wig.js',
+            './src/javascript/app/**/!(ng-wig|app).js'
+        ],
+        
+        proxies: {
+            '/': 'http://localhost:8888/'
```

Comment:
Not sure. I have copied bits of karma configuration file from existing repo. Do you have any clue or should I search for it?